maintainers/maintainer-list.nix

```diff
@@ -5849,6 +5849,12 @@
     github = "DavHau";
     githubId = 42246742;
   };
+  daviaaze = {
```

Comment:
This should be in first commit, called `maintainers: add daviaaze`
The package should be in second commit